### ISSUE TITLE
The cmsAddContest commnd can delete submissions

### ISSUE THREAD

**Issue by wil93** [cms-dev/cms#490]
To reproduce:

1. Run `cmsAddContest path` (it will add the participations and the contest).
2. Tie tasks to the contest using AWS.
3. Send some submissions using a valid participation.
4. Add some more users in contest.yaml.
5. Run: `cmsAddContest --update-contest path` (it should just add some more participations to the contest).
6. All the previous submissions are deleted from the database.

I suspect that this happens because in step 5 the participations are deleted and then created a second time (meanwhile, the ones that were added in step 4 are going to be created for the first time).

**Comment by lucach** [comment]
Am I missing something or ``cmsAddContest --update-contest`` is currently broken with the ``italy_yaml`` loader?

Base loader required a change with 5884ffd . 7b69d85 fixes the problem for polygon loader, but I can't find the change for the Italian loader.

In fact ``cmsAddContest --update-contest -L italy_yaml`` complains about the not-extended ``contest_has_changed()`` method.

**Comment by stefano-maggiolo** [comment]
I have to admit that I lost touch from all the importing features. It would be nice to have some proper tests on those to make sure that we don't introduce regressions like this, especially for italy_yaml. William or Luca, would you be available for writing such tests?

**Comment by wil93** [comment]
@lucach You're right :sweat_smile: I totally forgot that (in production) I manually added a dummy

```python
def contest_has_changed(self):
    return True
```

exactly to avoid the complaint you noticed. Then I went on using `--update-contest` (and I messed up, so I opened this issue).

So, yes, we should add `contest_has_changed` and `user_has_changed`, maybe as dummy methods for the time being. After that, I guess that the current issue will still be valid and should be fixed as well. I will be busy until the 26th, after that I can work on this, if no one will do it in the mean time.
